src/test/frontend/db/query_dsl_test.cljs

```diff
@@ -112,15 +113,27 @@ prop-d:: nada"}])
               (dsl-query "(property prop-d)")))
       "Blocks that have a property"))
 
-(deftest page-property-queries
+(deftest block-property-queries
```

Comment:
Since this option effects how properties are persisted, it is important to test that existing property filters work fine